.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Reviewers
+<!--- Call out additional reviewers -->
+<!--- in addition to the assigned reviewer -->
+<!--- example: @heyYou -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Related Issue
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
+<!--- Please link to the issue here: -->
+
+## Closes
+<!--- You can use "closes #42" to automatically -->
+<!--- close an issue ticket when the -->
+<!--- pull request is merged -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+
+## How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate):
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
let's get rid of the **CONTRIBUTING** line for now

I have executed makemigrations (if necessary)
I have created and pushed new docker images (if necessary)